bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Thing.java

```diff
@@ -23,9 +24,25 @@
  * through a {@link Bridge}.
  *
  * @author Dennis Nobel - Initial contribution and API
+ * @author Thomas Höfer - Added thing and thing type properties
  */
 public interface Thing {
 
+    /** the key for the vendor property */
```

Comment:
It was my suggestion. Actually there was no need for a separate enumeration class any more. For me the constants only makes sense at the thing, as they are part of the thing. We could also define a separate class like CommonThingPropertyConstants, but I don´t think that having an own class makes it more "unimportant".

I think these properties are important, because it is the agreed and predefined set by the framework, which a binding should define or add to a thing (if possible). And if you want to retrieve a "standard" property for a thing, I would expect the constant exactly at this point.

The question if every "important" property should be a separate method is valid (I also don´t want it). But I think we can live with a generic map here, because all properties have in common that there is logic in the framework for these properties. It is only a list of properties, maybe for the UI, maybe for the binding itself.